s3_data_management/s3_data_management.py

```diff
@@ -17,16 +17,16 @@
 def create_s3_client():
     """Creates a client that connects to s3 on AWS."""
     return client("s3",
-                  aws_access_key_id=environ['AWS_ACCESS_KEY_ID'],
-                  aws_secret_access_key=environ['AWS_SECRET_ACCESS_KEY'])
+                  aws_access_key_id=environ['AWS_ACCESS_KEY_ID_'],
+                  aws_secret_access_key=environ['AWS_SECRET_ACCESS_KEY_'])
 
 
 def get_bucket_keys(s3_client: client, folder_path: str, bucket_name: str) -> list:
     """Returns a list of keys of csv files, with a prefix matching the folder_path."""
     objects = s3_client.list_objects(
         Bucket=bucket_name, Prefix=folder_path).get('Contents')
     if objects:
-        return [obj['Key'] for obj in objects if '.csv' in obj['Key']]
+        return [obj['Key'] for obj in objects if obj['Key'].split('.')[-1] == 'csv']
```

Comment:
why this change?

Reply:
I was fiddling around earlier, but I don't think this actually makes a difference now

Reply:
No worries, I'll approve